src/DTO/UserDTO.php

```diff
@@ -146,6 +146,17 @@ class UserDTO
      */
     public $surname;
 
+    /**
+     * User biography
+     * @var string
+     * @Serializer\Type("string")
+     * @Assert\NotBlank(groups={"UserEdit"})
```

Comment:
No need of `@Assert\NotBlank(groups={"UserEdit"})`. Imho, biography should be optional